### ISSUE TITLE
Allowing more real estate for writing a note

### ISSUE THREAD

**Issue by henkekm** [BoostIO/boost-app-discussions#50]
Hi everyone. I'd like to propose the ability to remove the sidebars from view, and allow for full app-width writing of a selected note. Adjusting the width of each (the Folders sidebar, and the filter results sidebar) would be neat, but the main feature I'm seeking is maximizing the view of the note while writing.

Thank you kindly,
-Kevin

**Comment by Rokt33r** [comment]
I agree with that. I'll inform you here when each feature shipped.(I can start work this on Feb because of my semester exam)

![Challenge-Accepted](http://memesvault.com/wp-content/uploads/Challenge-Accepted-Barney-Stinson-06.jpg)
